### PR TITLE
Added documentation for new property in online meeting

### PR DESCRIPTION
**Pull request by frankpeng7**

### CODE REVIEW

File: api-reference/beta/resources/onlinemeeting.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks).
 | allowMeetingChat      | [meetingChatMode](#meetingchatmode-values) | Specifies the mode of meeting chat. |
 | allowTeamworkReactions | Boolean | Indicates if Teams reactions are enabled for the meeting. |
 | alternativeRecording  | Stream | The content stream of the alternative recording of a [Microsoft Teams live event](/microsoftteams/teams-live-events/what-are-teams-live-events). Read-only. |
+| anonymizeIdentityForRoles    | onlineMeetingRole collection | Specifies whose identity will be anonymized in meeting. Possible values are: `attendee`. Once `attendee` value is added, it cannot be removed via the Patch method.|
```
Comment (Danielabom) [inline api-reference/beta/resources/onlinemeeting.md:40]: Please consider using the following description: "Specifies whose identity will be anonymized in the meeting. Possible values are: `attendee`. The `attendee` value cannot be removed through a PATCH operation."

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @frankpeng7. I added one comment, please review it. Thanks!

**Review by Danielabom — APPROVED** [review]

File: api-reference/beta/resources/onlinemeeting.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks). S
 | allowMeetingChat      | [meetingChatMode](#meetingchatmode-values) | Specifies the mode of meeting chat. |
 | allowTeamworkReactions | Boolean | Indicates if Teams reactions are enabled for the meeting. |
 | alternativeRecording  | Stream | The content stream of the alternative recording of a [Microsoft Teams live event](/microsoftteams/teams-live-events/what-are-teams-live-events). Read-only. |
+| anonymizeIdentityForRoles    | onlineMeetingRole collection | Specifies whose identity will be anonymized in the meeting. Possible values are: `attendee`. The `attendee` value cannot be removed through a PATCH operation once added.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/onlinemeeting.md:40]: Please add this new property to the JSON representation section of this topic. 

In addition, I noticed the onlineMeetingRole evolvable enumeration has more values: presenter, unknownFutureValue, producer, coorganizer. As defined here: https://docs.microsoft.com/en-us/graph/api/resources/meetingparticipantinfo?view=graph-rest-beta#onlinemeetingrole-values
Would you be so kind to add them and include the disclaimer to add the appropiate header to access those values defined after unknownFutureValue, please?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @frankpeng7, 

Before we publish, please add the new property to the JSON representation section and include the rest of the possible values of the member with the note to add the appropiate header.

Thank you!

Isaac

File: api-reference/beta/resources/onlinemeeting.md (hunk 2 above)
Comment (frankpeng7) [reply api-reference/beta/resources/onlinemeeting.md:40]: I've added json representation. Despite the type of this property is "onlineMeetingRole", the only allowed value is "attendee" (other values such as "presenter", "producer" will not be applicable yet)

File: api-reference/beta/resources/onlinemeeting.md [hunk 3]
```diff
@@ -117,6 +118,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks). S
   "allowedPresenters": "String",
   "allowMeetingChat": {"@odata.type": "microsoft.graph.meetingChatMode"},
   "allowTeamworkReactions": "Boolean",
+  "anonymizeIdentityForRoles": "attendee",
```
Comment (isvargasmsft) [inline api-reference/beta/resources/onlinemeeting.md:121]: Please change this to a collection of Strings.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**